### PR TITLE
Updated the Secret Sealer document

### PR DESCRIPTION
**Pull request by darren-rafay**
Updated the Secret Sealer document. Please review and provide feedback.

I will work on the Pipeline document next and update when it is ready.

### CODE REVIEW

File: docs/resources/secretsealer.md [hunk 1]
```diff
@@ -8,69 +8,114 @@ description: |-
...
+  metadata {
+    name    = "tfdemosecretsealer1"
+    project = "terraform"
+  }
+  spec {
+    type = ""
+    sharing {
+      enabled = false
+    }
+    version = ""
```
Comment (kutumba-rafay) [inline docs/resources/secretsealer.md:27]: need to specify version

File: docs/resources/secretsealer.md [hunk 2]
```diff
@@ -8,69 +8,114 @@ description: |-
...
+
+```terraform
+#Create a secret sealer, sharing across project disabled
+resource "rafay_secretsealer" "tfdemosecretsealer1" {
+  metadata {
+    name    = "tfdemosecretsealer1"
+    project = "terraform"
+  }
+  spec {
+    type = ""
```
Comment (kutumba-rafay) [inline docs/resources/secretsealer.md:23]: need to specify type

File: docs/resources/secretsealer.md [hunk 3]
```diff
@@ -8,69 +8,114 @@ description: |-
...
+  }
+}
+
+resource "rafay_secretsealer" "tfdemosecretsealer2" {
+  metadata {
+    name    = "tfdemosecretsealer2"
+    project = "terraform"
+  }
+  spec {
+    type = ""
```
Comment (kutumba-rafay) [inline docs/resources/secretsealer.md:37]: need to specify type

File: docs/resources/secretsealer.md [hunk 4]
```diff
@@ -8,69 +8,114 @@ description: |-
...
+    sharing {
+      enabled = true
+      projects {
+        name = "defaultproject"
+      }
+    }
+    version = ""
```
Comment (kutumba-rafay) [inline docs/resources/secretsealer.md:44]: need to specify version

File: docs/resources/secretsealer.md [hunk 5]
```diff
@@ -8,69 +8,114 @@ description: |-
...
 ### Nested Schema for `spec`
 
-Optional:
+***Optional***
 
-- `sharing` (Block List, Max: 1) secret sealer sharing configuration (see [below for nested schema](#nestedblock--spec--sharing))
-- `type` (String) secret sealer type
-- `version` (String) secret sealer sharing configuration
+- `sharing` - (Block List, Max: 1) Enables sharing the pipeline. (See [below for nested schema](#nestedblock--spec--sharing))
+- `type` - (String) The type of secret sealer.
```
Comment (kutumba-rafay) [inline docs/resources/secretsealer.md:87]: Need to specify accepted values for type

File: docs/resources/secretsealer.md [hunk 6]
```diff
@@ -8,69 +8,114 @@ description: |-
...
+
 ### Nested Schema for `spec`
 
-Optional:
+***Optional***
 
-- `sharing` (Block List, Max: 1) secret sealer sharing configuration (see [below for nested schema](#nestedblock--spec--sharing))
-- `type` (String) secret sealer type
-- `version` (String) secret sealer sharing configuration
+- `sharing` - (Block List, Max: 1) Enables sharing the pipeline. (See [below for nested schema](#nestedblock--spec--sharing))
```
Comment (kutumba-rafay) [inline docs/resources/secretsealer.md:86]: it should be secret sealer, not pipeline.

**Review by kutumba-rafay — CHANGES_REQUESTED** [review]

File: docs/resources/secretsealer.md [hunk 7]
```diff
@@ -8,69 +8,114 @@ description: |-
...
+***Optional***
 
-- `annotations` (Map of String) annotations of the resource
-- `description` (String) description of the resource
-- `labels` (Map of String) labels of the resource
-- `name` (String) name of the resource
-- `project` (String) Project of the resource
+- `annotations` - (Map of String) Annotations of the resource.
+- `description` - (String) Description of the resource.
+- `labels` - (Map of String) Labels of the resource.
```
Comment (hardik-rafay) [inline docs/resources/secretsealer.md:76]: Remove labels & annotations.

**Review by hardik-rafay — CHANGES_REQUESTED** [review]

**Review by hardik-rafay — APPROVED** [review]

**Pull request closed without merging**